mantidproject/mantid
# Add switchable mutli-period columns to ISIS SANS GUI

Fixes #20748

This PR adds the option to dynamically add or remove the multi-period columns in the data table in the new ISIS SANS GUI. This has been requested by Steve King. Since the vast majority of reductions don't make use of a multi-period set-up, it makes sense to have these columns off by default.

# To test:

- Open the sans v2 GUI
- Get the SANS data from _\\olympic\Babylon5\Scratch\Matthew Andrew\GUI_TEST_
- Check that the Multi period box below the table is initially unselected.
- Select it and check that the period columns appear.
- With the box unselected load a user file and batch file, click process.
- Try the same thing with the box selected. 
- Code Review

**Release Notes** 

Have added to release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
